Adding changelog for evaluation rename and flows (#97)
humanloop/humanloop-docs: fern/changelog/2024-09-16.md

## Evaluation Names

You can now name your Evaluations in the UI and via the API. This is helpful for more easily identifying the purpose of your different Evaluations, especially when multiple teams are running different experiments.

> ![Evaluation with a name](../assets/images/changelogs/evaluation-rename.png)

Reviewer: @gavinfd-hl need a better screenshot here. This only has 2 logs and a single version, makes the app look bad i think

Author: Feels slightly arbitrary at that point but okay I'll create a different one.

> In the API, pass in the `name` field when creating your Evaluation to set the name. Note that names must be unique for all Evaluations for a specific file. In the UI, navigate to your Evaluatio,n and you will see an option to rename it in the header.

Reviewer 2: ```suggestion
In the API, pass in the `name` field when creating your Evaluation to set the name. Note that names must be unique for all Evaluations for a specific file. In the UI, navigate to your Evaluation and you will see an option to rename it in the header.
```

## Introducing Flows

> We've added a new key building block to our app with the first release of Flows. Flows allow you to collect multiple Logs into a Trace and evaluate them together. Each Trace aims to represent one interaction between end user and your system, offering better observability.

Reviewer 2: ```suggestion
We've added a new key building block to our app with the first release of Flows. Flows allow you to collect multiple Logs into a Trace and evaluate them together. Each Trace should to represent one interaction between an end-user and your system, offering better observability.
```

> We've added a new key building block to our app with the first release of Flows. Flows allow you to collect multiple Logs into a Trace and evaluate them together. Each Trace should to represent one interaction between an end-user and your system, offering better observability.

Reviewer 2: ```suggestion
We've added a new key building block to our app with the first release of Flows. Flows allow you to collect multiple Logs into a Trace and evaluate them together, improving observability across your apps.
```

> This release focuses on improving the code first workflows for Evaluating more complex AI applications like RAG and Agent based apps. Currently, only the inputs and output of a Trace can be evaluated, with a future release allowing Evaluators access to all Logs inside the Trace. Additionally, we will build on this by adding UI first visualisations and management of your Flows, and offer SDK equivalents to the new API resource.

Reviewer 2: ```suggestion
This release focuses on improving the code-first workflows for evaluating more complex AI applications like RAG and Agent-based apps. Currently, only the inputs and output of a Trace can be evaluated, with a future release allowing Evaluators access to all Logs inside the Trace. Additionally, we will build on this by adding UI-first visualisations and management of your Flows, and offer SDK equivalents to the new API resource.
```

i find the quick mention of limitations to be jarring and the "we will build on this by adding..." bit to be very me-focused as opposed to "we will make it easier for you to ..."

> This release focuses on improving the code-first workflows for evaluating more complex AI applications like RAG and Agent-based apps. Currently, only the inputs and output of a Trace can be evaluated, with a future release allowing Evaluators access to all Logs inside the Trace. Additionally, we will build on this by adding UI-first visualisations and management of your Flows, and offer SDK equivalents to the new API resource.
> 
> Sessions will be discontinued in the favour of Flows in the near future. New Sessions can no longer be created, but existing ones can still be viewed. Reach out to us for guidance on how to migrate your Session-based workflows to Flows.

Reviewer 2: ```suggestion
This release focuses on improving the code-first workflows for evaluating more complex AI applications like RAG and Agent-based apps. 

You can start logging your app traces with the [Flows log endpoint](https://humanloop.com/docs/v5/api-reference/flows/log).
(Check back later for a more in-depth tutorial we'll be releasing shortly.)

### What's next

We'll soon be extending support for evaluating Traces, allowing Evaluators to access all Logs inside a Trace. 
Additionally, we will build on this by adding UI-first visualisations and management of your Flows.

We'll sunset Sessions in favour of Flows in the near future. Reach out to us for guidance on how to migrate your Session-based workflows to Flows.
```

> Sessions will be discontinued in the favour of Flows in the near future. New Sessions can no longer be created, but existing ones can still be viewed. Reach out to us for guidance on how to migrate your Session-based workflows to Flows.

Reviewer 2: ```suggestion
Sessions will be discontinued in favour of Flows in the near future. Reach out to us for guidance on how to migrate your Session-based workflows to Flows.
```

I'd add this as an addendum at the bottom tbh. like clearly demarcated as "not part of the main changelog"
